api/renter.go

```diff
@@ -83,9 +99,32 @@ type (
 
 // renterHandlerGET handles the API call to /renter.
 func (api *API) renterHandlerGET(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
+	// calculate financial metrics from contracts
+	periodStart := api.renter.CurrentPeriod()
+	var fm RenterFinancialMetrics
+	for _, c := range api.renter.Contracts() {
+		if c.StartHeight < periodStart {
+			continue
+		}
+		fm.ContractSpending = fm.ContractSpending.Add(c.TotalCost)
+		fm.DownloadSpending = fm.DownloadSpending.Add(c.DownloadSpending)
+		fm.UploadSpending = fm.UploadSpending.Add(c.UploadSpending)
+		fm.StorageSpending = fm.StorageSpending.Add(c.StorageSpending)
+		fm.Unspent = fm.Unspent.Add(c.RenterFunds())
+	}
+	settings := api.renter.Settings()
+	fm.Unspent = fm.Unspent.Add(settings.Allowance.Funds)
+	if fm.Unspent.Cmp(fm.ContractSpending) < 0 {
+		// avoid negative currency error
+		fm.Unspent = types.ZeroCurrency
+	} else {
+		fm.Unspent = fm.Unspent.Sub(fm.ContractSpending)
+	}
```

Comment:
oh wait I see.

you add the unspent of each contract, then you add all of the funds of the allowance, then finally you subtract out the total spent on contracts.

Ok, that does seem correct. But I think it would help if there were more comments explaining that.